README.md

```diff
@@ -75,37 +75,44 @@ Usage: ruby ticketmatch.rb [options]
     -h, --help                       this message
 ```
 
-# Batch processing
+# Batch processing for puppet-agent and its components
```

Comment:
(since this only applies to puppet-agent)

Reply:
(that's why we named it **pa_**matchbatch.sh :))